snap/README.md

```diff
@@ -60,60 +93,45 @@ started.
 
 The following syntax is used to specify service-specific configuration overrides:
 
-```env.<stanza>.<config option>```
+```
+env.<stanza>.<config option>
+```
 
 For instance, to setup an override of the service's Port use:
 
-```$ sudo snap set env.service.port=2112```
-
-And restart the service:
-
-```$ sudo snap restart edgex-app-service-configurable```
-
-**Note** - at this time changes to configuration values in the [Writable] section are not supported.
-
-For details on the mapping of configuration options to Config options, please refer to "Service Environment Configuration Overrides".
-
-### Startup environment variables
-
-EdgeX services by default wait 60s for dependencies (e.g. Core Data) to become available, and will exit after this time if the dependencies aren't met. The following options can be used to override this startup behavior on systems where it takes longer than expected for the dependent services provided by the edgexfoundry snap to start. Note, both options below are specified as a number of seconds.
-    
-To change the default startup duration (60 seconds), for a service to complete the startup, aka bootstrap, phase of execution by using the following command:
-
 ```bash
-$ sudo snap set edgex-app-service-configurable startup-duration=60
+sudo snap set env.service.port=2112
 ```
 
-The following environment variable overrides the retry startup interval or sleep time before a failure is retried during the start-up, aka bootstrap, phase of execution by using the following command:
+And restart the service:
 
 ```bash
-$ sudo snap set edgex-app-service-configurable startup-interval=1
+sudo snap restart edgex-app-service-configurable
 ```
 
-**Note** - Should the environment variables be modified after the service has started, the service must be restarted.
+**Note** - at this time changes to configuration values in the [Writable] section are not supported.
+
+For details on the mapping of configuration options to Config options, please refer to [Service Environment Configuration Overrides](#service-environment-configuration-overrides) below.
 
 ### Profiles
 There are a number of sub-directories in the snap's configuration directory, these sub-directories are referred to as profiles. Before you can start the service, you must select one of them, using the `snap set` command (or via snapd's REST API). For example, to use the mqtt-export profile you would run:
 
-```
-$ sudo snap set edgex-app-service-configurable profile=mqtt-export
+```bash
+sudo snap set edgex-app-service-configurable profile=mqtt-export
 ```
 
 In addition to instructing the service to read a different configuration file, the profile will also be used to name the service when it registers itself to the system.
-
-**Note** - currently the only two profiles available for Ireland (2.0) are http-export and mqtt-export.
+For details on currently available profiles, please see [here](https://github.com/edgexfoundry/app-service-configurable/tree/main/res).
```

Comment:
use relative links to respect the versioning.

Reply:
Updated.